doc/source/user_guide/adaptive_templates.rst

```diff
@@ -428,6 +428,39 @@ the function ``draw_objects_debug()`` can be used to plot the regions:
 
     Plot of rotor pocket regions drawn using the ``draw_objects()`` function.
 
+Adding imported DXF geometries to adaptive templates
+****************************************************
+Custom geometry can be imported to Motor-CAD from a DXF file. For details on how to import a custom
+geometry from DXF file, please see the 'Custom Machine Geometries' tutorial supplied with Motor-CAD.
+
+Once a custom geometry has been imported, it is automatically separated into regions which will
+appear under 'Import' in the Geometry -> Editor tab. These imported geometry regions can be
+accessed using ``get_region_dxf()`` from ``ansys.motorcad.core``. The geometry regions that are
+currently set in the Motor-CAD model are shown under 'Template' in the tree.
+
+.. figure:: ../images/Adaptive_Geometry_DXF__1.png
+    :width: 500pt
+
+    Geometry -> Editor -> Geometry tab with an imported custom geometry from DXF file.
```

Comment:
Is this line supposed to be a figure caption option or an alternative attribute for accessibility? This isn't a compete sentence if it is paragraph content, UI elements are to be formatted in bold.

Reply:
It's a figure caption. I've reformatted the UI elements in bold